mlflow/sklearn/utils.py

```diff
@@ -103,6 +118,248 @@ def _get_args_for_score(score_func, fit_func, fit_args, fit_kwargs):
+    has sample_weight), extract it from fit_args or fit_kwargs as
+    (y_true, y_pred, ...... sample_weight), otherwise as (y_true, y_pred, ......)
+    3. return a dictionary of metric(name, value)
+
+    :param trained_estimator: The already fitted classifier
+    :param fit_args: Positional arguments given to fit_func.
+    :param fit_kwargs: Keyword arguments given to fit_func.
+    :return: dictionary of (function name, computed value)
+    """
+    import sklearn
+
+    fit_arg_names = _get_arg_names(trained_estimator.fit)
+    X, y_true, y_pred = _get_samples_labels_and_predictions(
+        trained_estimator, fit_args, fit_kwargs, fit_arg_names
+    )
+    sample_weight = (
+        _get_sample_weight(fit_arg_names, fit_args, fit_kwargs)
+        if _SAMPLE_WEIGHT in fit_arg_names
+        else None
+    )
+
+    classifier_metrics = [
+        _SklearnMetric(
+            name="precision_score",
+            function=sklearn.metrics.precision_score,
+            arguments=dict(
+                y_true=y_true, y_pred=y_pred, average="weighted", sample_weight=sample_weight
+            ),
+        ),
+        _SklearnMetric(
+            name="recall_score",
+            function=sklearn.metrics.recall_score,
+            arguments=dict(
+                y_true=y_true, y_pred=y_pred, average="weighted", sample_weight=sample_weight
+            ),
+        ),
+        _SklearnMetric(
+            name="f1_score",
+            function=sklearn.metrics.f1_score,
+            arguments=dict(
+                y_true=y_true, y_pred=y_pred, average="weighted", sample_weight=sample_weight
+            ),
+        ),
+        _SklearnMetric(
+            name="accuracy_score",
+            function=sklearn.metrics.accuracy_score,
+            arguments=dict(
+                y_true=y_true, y_pred=y_pred, normalize=True, sample_weight=sample_weight
+            ),
+        ),
+    ]
+
+    if hasattr(trained_estimator, "predict_proba"):
+        y_pred_proba = trained_estimator.predict_proba(X)
+
+        classifier_metrics.extend(
+            [
+                _SklearnMetric(
+                    name="log_loss",
+                    function=sklearn.metrics.log_loss,
+                    arguments=dict(y_true=y_true, y_pred=y_pred_proba, sample_weight=sample_weight),
+                ),
+                _SklearnMetric(
+                    name="roc_auc_score",
+                    function=sklearn.metrics.roc_auc_score,
+                    arguments=dict(
+                        y_true=y_true,
+                        y_score=y_pred_proba,
+                        average="weighted",
+                        sample_weight=sample_weight,
+                        multi_class="ovo",
+                    ),
+                ),
+            ]
+        )
+
+    return _get_metrics_value_dict(classifier_metrics)
+
+
+def _get_regressor_metrics(trained_estimator, fit_args, fit_kwargs):
```

Comment:
```suggestion
def _get_regressor_metrics(fitted_estmator, fit_args, fit_kwargs):
```

to match sklearn's terminology?